docs/Toy/Toy_Ch2.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
<p>The formatting in this section is a bit weird. Is this intended or just a fragment of your import procedure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/739/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>What do you mean by formatting? It's mostly text from the MLIR tutorial</p><p> </p>

Reply:
There are some linebreaks that look weird to me.

Reply:
Can you give some examples? I don't see anything weird

Reply:
Same. I guess it was a ReviewNB problem?  Anyway, you can ignore this comment :)